bff/values.yaml

```diff
@@ -5,9 +5,12 @@
 replicaCount: 1
 
 image:
-  repository: bff
-  tag: 1471
-  pullPolicy: Always
+  repositoryBff: bff
+  tagBff: 1471
+  pullPolicyBff: Always
+  repositorySidecar: bff
+  tagSidecar: 1471
+  pullPolicySidecar: Always
```

Comment:
If we separate all the new `sidecar` properties into a specific `.Values.sidecar` group, this won't be needed anymore